### DISCUSSION TITLE
[WIP] The MINI-Craft

### DISCUSSION THREAD

**Discussion by eXponenta** [aduros/wasm4#63] [General]
Follow my game!
Developing in progress.

https://github.com/exponenta/wasm4-game

Plans:
 - Big random generated world! <<! DONE
 - Craft system!
 - Story and tasks!

**Comment by aduros** [comment]
Whoa, you're making an open world RPG? Sounds awesome!

It's looking great so far, I like the character animation. Even kicking trees is satisfying :smile:

**Reply by eXponenta** [reply]
Yep, i already introduce a random generated world. You can move between chunks that generated on fly.
